wayshot/src/cli.rs

```diff
@@ -1,43 +1,46 @@
-use std::path::PathBuf;
-
-use clap::arg;
-
-use clap::Parser;
-use eyre::WrapErr;
-
 use crate::utils::EncodingFormat;
-use clap::builder::TypedValueParser;
+use clap::Parser;
+use std::path::PathBuf;
+use tracing::Level;
 
 #[derive(Parser)]
 #[command(version, about)]
 pub struct Cli {
```

Comment:
I feel like we should have a look at this crate: https://crates.io/crates/clap-serde-derive

It merges the serde and CLI config and would mean that we don't have options here all over the place.